Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu
+MAINTAINER Kaha Team
+LABEL Description="Image for kaha: quick relief lookup for nepal earthquake"
+
+# install node
+ADD http://nodejs.org/dist/v0.12.2/node-v0.12.2-linux-x64.tar.gz /tmp/
```

Comment:
Might replace with wget/curl which is what docker docs recommend, but this was easier.